src/config.js

```diff
@@ -3,31 +3,39 @@ import path from 'path'
 import fs from 'fs'
 import os from 'os'
 import isDev from 'electron-is-dev'
-import {app} from 'electron'
+import {app, dialog} from 'electron'
 import FileHistory from './utils/file-history'
 
-export const logoIpfsIce = (() => {
-  const p = path.resolve(path.join(__dirname, 'img'))
-
-  if (os.platform() === 'darwin') {
-    return path.join(p, 'icons/ice.png')
-  }
+// Set up what to do on Uncaught Exceptions
+process.on('uncaughtException', (error) => {
+  const msg = error.message || error
+  logger.error(`Uncaught Exception: ${msg}`, error)
+  dialog.showErrorBox('Uncaught Exception:', msg)
+  process.exit(1)
+})
 
-  return path.join(p, 'ipfs-logo-ice.png')
-})()
+// Set up crash reporter or electron debug
+if (!isDev) {
+  require('electron').crashReporter.start({
+    productName: 'Station',
+    companyName: 'IPFS',
+    submitURL: 'https://ipfs.io',
```

Comment:
What is this submitting to?

Reply:
This is submitting nothing to nowhere. I'll remove this. It seems to be here since the 'old days' 😛